mission-vision-values.md

```diff
@@ -0,0 +1,100 @@
+requirement for excellent observability, it’s still unreasonable and
+unrealistic to expect developers of application software to add or maintain the
+necessary instrumentation on their own. That is a massive undertaking, and in
+practice it’s not “just code” – there is also necessary alignment around
+protocols and semantic conventions for tags, attributes, and other metadata to
+consider.
+
+So how do we get high-quality, turnkey telemetry without a massive,
+unsustainable engineering effort? This is where OpenTelemetry comes in. To
+achieve our vision, OpenTelemetry sees five key opportunities, listed here:
+
+**Opportunity I: Telemetry should be easy.** With OpenTelemetry we want
+high-quality telemetry to be easy, especially for end-users. That means that
+OpenTelemetry should have fast time-to-value, set reasonable defaults yet allow
+for customization, and pair with excellent documentation and a top-tier overall
+developer experience.
+
+**Opportunity II: Telemetry should be universal.** Telemetry protocols and
+conventions should be unified across languages and signal types (tracing,
+metrics, logging, etc), not divergent or siloed. This means that OpenTelemetry
+aspires to find technical solutions that work consistently, both locally and
+globally.
+
+**Opportunity III: Telemetry should be vendor-neutral.** For decades, monitoring
+and observability vendors have created differentiation – and lock-in – through
+agents that automatically gather useful telemetry from a wide integration
+surface. This lock-in is limiting for customers and inhibits innovation by
+tightly coupling telemetry collection and telemetry storage and analysis. With
+OpenTelemetry, we strive to provide a level playing field for all observability
+providers, avoiding lock-in for any vendor and interoperating with other OSS
+projects in the telemetry and observability ecosystem.
+
+**Opportunity IV: Telemetry should be loosely coupled.** OpenTelemetry end-users
+should be able to pick and choose from the pieces they want without bringing in
+the rest of the project. To enable this, OpenTelemetry’s software architecture
+is decoupled wherever possible. As a corollary, this also means that
+OpenTelemetry does not want to “pick winners” when it comes to particular
+projects or technologies: where possible, we prefer to create optionality for
+our end-users.
+
+**Opportunity V: Telemetry should be built-in.** Historically, telemetry was
+something developers integrated manually or via post-compilation agents.
+OpenTelemetry believes that high-quality telemetry can be built in to the
+entire software stack – just like testing has been for decades.
+
+While the structure and technical details of OpenTelemetry may change over
+time, these five key opportunities will remain outstanding until we achieve our
+mission, and as a project we refer to them to orient – and reorient – as we
+chart our path.
+
+## Engineering Values (the principles that guide our contributions)
```

Comment:
These values describe the artifacts we produce. Do we also want to document values we hold as an engineering community/organization? E.g.,
* bias towards action and experimentation vs consensus-building
* inclusion, mentorship and investment in new contributors
* respect for maintainers

Reply:
these values were covered in https://docs.google.com/document/d/1L92ThlgvSxYlQixpBNEovz8NnyOZfKu1E3zTT2gxaCY/edit#heading=h.id952e4ms1v7 under execution principles. Including those may be too much for this document. Do you feel this have to be formulated as values?

Reply:
Yeah, my concern was brevity. I think it's a great idea to document "the way we work", but my thought process was that "the way we work" might change significantly over time as the project matures, and as such it doesn't make it into this doc which is meant to be for more "permanent" / unchanging values. Thoughts?

Reply:
If that material will go to a different, prominent location then I'm 👍🏽